package.json

```diff
@@ -4,7 +4,8 @@
   "description": "A minimal core of the Infragram project in JavaScript.",
   "main": "dist/infragram.js",
   "scripts": {
-    "build": "grunt build"
+    "build": "grunt build",
+    "test": "jest"
```

Comment:
Hi @Forchapeatl I think I have this line wrong. When you ran `npm test` i would have guessed it to run this. But maybe I'm wrong and this line was not needed? In any case, can you see how we're actually running in GitHub Actions now, and maybe make a suggestion of how to fix this? Thank you!!! Exciting!!

Reply:
The error I see in the log is: 

```
> infragram@0.3.0 test /home/runner/work/infragram/infragram
> jest

/home/runner/work/infragram/infragram/node_modules/jest-cli/build/cli/index.js:161
    if (error?.stack) {
              ^

SyntaxError: Unexpected token .
    at Module._compile (internal/modules/cjs/loader.js:[7](https://github.com/publiclab/infragram/runs/8025926511?check_suite_focus=true#step:7:8)23:23)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:7[8](https://github.com/publiclab/infragram/runs/8025926511?check_suite_focus=true#step:7:9)[9](https://github.com/publiclab/infragram/runs/8025926511?check_suite_focus=true#step:7:10):[10](https://github.com/publiclab/infragram/runs/8025926511?check_suite_focus=true#step:7:11))
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:[12](https://github.com/publiclab/infragram/runs/8025926511?check_suite_focus=true#step:7:13))
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/home/runner/work/infragram/infragram/node_modules/jest-cli/build/index.js:[13](https://github.com/publiclab/infragram/runs/8025926511?check_suite_focus=true#step:7:14):12)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
npm ERR! Test failed.  See above for more details.
Error: Process completed with exit code 1.
```

Here: https://github.com/publiclab/infragram/runs/8025926511?check_suite_focus=true#step:7:10

Reply:
@jywarren , sorry for the late response. I've been trying to compile some documents from school.

https://github.com/publiclab/infragram/blob/1dabd5ceee543ed02985b1c60b7d841bbf92ac94/.github/workflows/tests.yml#L21

could we do a simple `npm install` instead ? I think jest is not being installed properly.

Reply:
I think npm install is automatically run beforehand; that line is only because grunt-cli is not installed as a regular dependency. Previously i was getting something like "no command jest" but now i think jest is being run. 

But, now that I look at this: 

https://github.com/publiclab/image-sequencer/blob/main/.github/workflows/tests.yml#L74-L75

I think you could be right. I'll try adding `grunt-cli` to the devDependencies and then modifying that line to just be `npm install`.